{
  "type": "issue",
  "issue": {
    "id": 1837587765,
    "node_id": "I_kwDOABII585th101",
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221",
    "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
    "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221/labels%7B/name%7D",
    "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221/comments",
    "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221/events",
    "html_url": "https://github.com/bitcoin/bitcoin/issues/28221",
    "number": 28221,
    "state": "closed",
    "state_reason": "completed",
    "title": "Flaky `wallet_transactiontime_rescan.py --legacy-wallet` functional test",
    "body": "Failed in CI on unrelated PR https://github.com/bitcoin/bitcoin/pull/28216.\r\n\r\n```\r\n test  2023-08-04T17:39:25.417000Z TestFramework (ERROR): Called Process failed with 'error: timeout on transient error: Could not connect to the server 127.0.0.1:20800 (error code 0 - \"timeout reached\") \r\n                                   Make sure the bitcoind server is running and that you are connecting to the correct RPC port.\r\n                                   '\r\n                                   Traceback (most recent call last):\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 131, in main\r\n                                       self.run_test()\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/wallet_transactiontime_rescan.py\", line 209, in run_test\r\n                                       minernode.cli(\"-rpcwallet=encrypted_wallet\").walletpassphrase(\"passphrase\", 1)\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_node.py\", line 714, in __call__\r\n                                       return self.cli.send_cli(self.command, *args, **kwargs)\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_node.py\", line 779, in send_cli\r\n                                       raise subprocess.CalledProcessError(returncode, self.binary, output=cli_stderr)\r\n                                   subprocess.CalledProcessError: Command '/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-i686-pc-linux-gnu/src/bitcoin-cli' returned non-zero exit status 1.\r\n test  2023-08-04T17:39:25.614000Z TestFramework (DEBUG): Closing down network thread \r\n test  2023-08-04T17:39:25.665000Z TestFramework (INFO): Stopping nodes \r\n test  2023-08-04T17:39:25.665000Z TestFramework.node0 (DEBUG): Stopping node \r\n test  2023-08-04T17:39:25.665000Z TestFramework.node0 (ERROR): Unable to stop node. \r\n                                   Traceback (most recent call last):\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_node.py\", line 346, in stop_node\r\n                                       self.stop(wait=wait)\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/coverage.py\", line 50, in __call__\r\n                                       return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/authproxy.py\", line 127, in __call__\r\n                                       response, status = self._request('POST', self.__url.path, postdata.encode('utf-8'))\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/authproxy.py\", line 105, in _request\r\n                                       self.__conn.request(method, path, postdata, headers)\r\n                                     File \"/usr/lib64/python3.9/http/client.py\", line 1285, in request\r\n                                       self._send_request(method, url, body, headers, encode_chunked)\r\n                                     File \"/usr/lib64/python3.9/http/client.py\", line 1296, in _send_request\r\n                                       self.putrequest(method, url, **skips)\r\n                                     File \"/usr/lib64/python3.9/http/client.py\", line 1122, in putrequest\r\n                                       raise CannotSendRequest(self.__state)\r\n                                   http.client.CannotSendRequest: Request-sent\r\n test  2023-08-04T17:39:25.666000Z TestFramework.node1 (DEBUG): Stopping node \r\n```\r\nCI run: https://github.com/bitcoin/bitcoin/pull/28216/checks?check_run_id=15628645597 (https://cirrus-ci.com/task/5356802372534272?logs=ci#L2440).",
    "user": {
      "login": "darosior",
      "id": 22457751,
      "node_id": "MDQ6VXNlcjIyNDU3NzUx",
      "avatar_url": "https://avatars.githubusercontent.com/u/22457751?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/darosior",
      "html_url": "https://github.com/darosior",
      "followers_url": "https://api.github.com/users/darosior/followers",
      "following_url": "https://api.github.com/users/darosior/following%7B/other_user%7D",
      "gists_url": "https://api.github.com/users/darosior/gists%7B/gist_id%7D",
      "starred_url": "https://api.github.com/users/darosior/starred%7B/owner%7D%7B/repo%7D",
      "subscriptions_url": "https://api.github.com/users/darosior/subscriptions",
      "organizations_url": "https://api.github.com/users/darosior/orgs",
      "repos_url": "https://api.github.com/users/darosior/repos",
      "events_url": "https://api.github.com/users/darosior/events%7B/privacy%7D",
      "received_events_url": "https://api.github.com/users/darosior/received_events",
      "type": "User",
      "site_admin": false,
      "patch_url": null
    },
    "labels": [
      {
        "id": 62963516,
        "node_id": "MDU6TGFiZWw2Mjk2MzUxNg==",
        "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/Tests",
        "name": "Tests",
        "color": "d4c5f9",
        "default": false
      }
    ],
    "assignees": [],
    "author_association": "MEMBER",
    "locked": true,
    "comments": 9,
    "closed_at": "2023-09-05T12:00:52Z",
    "closed_by": {
      "login": "maflcko",
      "id": 6399679,
      "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
      "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/maflcko",
      "html_url": "https://github.com/maflcko",
      "followers_url": "https://api.github.com/users/maflcko/followers",
      "following_url": "https://api.github.com/users/maflcko/following%7B/other_user%7D",
      "gists_url": "https://api.github.com/users/maflcko/gists%7B/gist_id%7D",
      "starred_url": "https://api.github.com/users/maflcko/starred%7B/owner%7D%7B/repo%7D",
      "subscriptions_url": "https://api.github.com/users/maflcko/subscriptions",
      "organizations_url": "https://api.github.com/users/maflcko/orgs",
      "repos_url": "https://api.github.com/users/maflcko/repos",
      "events_url": "https://api.github.com/users/maflcko/events%7B/privacy%7D",
      "received_events_url": "https://api.github.com/users/maflcko/received_events",
      "type": "User",
      "site_admin": false,
      "patch_url": null
    },
    "created_at": "2023-08-05T05:36:49Z",
    "updated_at": "2025-02-27T07:34:53Z"
  },
  "events": [
    {
      "event": "commented",
      "id": 1666789562,
      "node_id": "IC_kwDOABII585jWTC6",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1666789562",
      "actor": {
        "login": "maflcko",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/maflcko",
        "html_url": "https://github.com/maflcko",
        "followers_url": "https://api.github.com/users/maflcko/followers",
        "following_url": "https://api.github.com/users/maflcko/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/maflcko/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/maflcko/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/maflcko/subscriptions",
        "organizations_url": "https://api.github.com/users/maflcko/orgs",
        "repos_url": "https://api.github.com/users/maflcko/repos",
        "events_url": "https://api.github.com/users/maflcko/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/maflcko/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2023-08-06T09:27:36Z",
      "updated_at": "2023-08-06T09:27:36Z",
      "author_association": "MEMBER",
      "body": "Looks like it did got dispatched to thread 2, but thread 0 was blocking for 24 minutes:\r\n\r\n```\r\n test  2023-08-04T17:19:25.474000Z TestFramework.bitcoincli (DEBUG): Running bitcoin-cli ['-rpcwallet=encrypted_wallet', 'walletpassphrase', 'passphrase', '1'] \r\n node0 2023-08-04T17:19:25.478233Z (mocktime: 2023-09-03T17:11:37Z) [http] [httpserver.cpp:254] [http_request_cb] [http] Received a POST request for /wallet/encrypted_wallet from 127.0.0.1:56776 \r\n node0 2023-08-04T17:19:25.478352Z (mocktime: 2023-09-03T17:11:37Z) [httpworker.2] [rpc/request.cpp:181] [parse] [rpc] ThreadRPCServer method=walletpassphrase user=__cookie__ \r\n...\r\n node0 2023-08-04T17:23:35.739266Z (mocktime: 2023-09-03T17:11:37Z) [httpworker.0] [wallet/scriptpubkeyman.h:254] [WalletLogPrintf] [encrypted_wallet] keypool added 400 keys (0 internal), size=800 (400 internal) \r\n...\r\n node0 2023-08-04T17:26:24.111206Z (mocktime: 2023-09-03T17:11:37Z) [scheduler] [net.cpp:1522] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  7744ms \r\n...\r\n node0 2023-08-04T17:43:54.247168Z (mocktime: 2023-09-03T17:11:37Z) [httpworker.0] [wallet/wallet.h:895] [WalletLogPrintf] [encrypted_wallet] Rescan completed in         1468844ms ",
      "user": {
        "login": "maflcko",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/maflcko",
        "html_url": "https://github.com/maflcko",
        "followers_url": "https://api.github.com/users/maflcko/followers",
        "following_url": "https://api.github.com/users/maflcko/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/maflcko/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/maflcko/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/maflcko/subscriptions",
        "organizations_url": "https://api.github.com/users/maflcko/orgs",
        "repos_url": "https://api.github.com/users/maflcko/repos",
        "events_url": "https://api.github.com/users/maflcko/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/maflcko/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/28221#issuecomment-1666789562",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221"
    },
    {
      "event": "commented",
      "id": 1666789809,
      "node_id": "IC_kwDOABII585jWTGx",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1666789809",
      "actor": {
        "login": "maflcko",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/maflcko",
        "html_url": "https://github.com/maflcko",
        "followers_url": "https://api.github.com/users/maflcko/followers",
        "following_url": "https://api.github.com/users/maflcko/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/maflcko/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/maflcko/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/maflcko/subscriptions",
        "organizations_url": "https://api.github.com/users/maflcko/orgs",
        "repos_url": "https://api.github.com/users/maflcko/repos",
        "events_url": "https://api.github.com/users/maflcko/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/maflcko/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2023-08-06T09:28:31Z",
      "updated_at": "2023-08-06T09:28:31Z",
      "author_association": "MEMBER",
      "body": "Probably an upstream GCE outage?",
      "user": {
        "login": "maflcko",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/maflcko",
        "html_url": "https://github.com/maflcko",
        "followers_url": "https://api.github.com/users/maflcko/followers",
        "following_url": "https://api.github.com/users/maflcko/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/maflcko/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/maflcko/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/maflcko/subscriptions",
        "organizations_url": "https://api.github.com/users/maflcko/orgs",
        "repos_url": "https://api.github.com/users/maflcko/repos",
        "events_url": "https://api.github.com/users/maflcko/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/maflcko/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/28221#issuecomment-1666789809",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221"
    },
    {
      "event": "commented",
      "id": 1706483185,
      "node_id": "IC_kwDOABII585ltt3x",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1706483185",
      "actor": {
        "login": "fanquake",
        "id": 863730,
        "node_id": "MDQ6VXNlcjg2MzczMA==",
        "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fanquake",
        "html_url": "https://github.com/fanquake",
        "followers_url": "https://api.github.com/users/fanquake/followers",
        "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
        "organizations_url": "https://api.github.com/users/fanquake/orgs",
        "repos_url": "https://api.github.com/users/fanquake/repos",
        "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fanquake/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2023-09-05T11:56:38Z",
      "updated_at": "2023-09-05T11:56:38Z",
      "author_association": "MEMBER",
      "body": "This is continuing to happen, i.e https://github.com/bitcoin/bitcoin/actions/runs/6082441277/job/16500239670.",
      "user": {
        "login": "fanquake",
        "id": 863730,
        "node_id": "MDQ6VXNlcjg2MzczMA==",
        "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fanquake",
        "html_url": "https://github.com/fanquake",
        "followers_url": "https://api.github.com/users/fanquake/followers",
        "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
        "organizations_url": "https://api.github.com/users/fanquake/orgs",
        "repos_url": "https://api.github.com/users/fanquake/repos",
        "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fanquake/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/28221#issuecomment-1706483185",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221"
    },
    {
      "event": "commented",
      "id": 1706484048,
      "node_id": "IC_kwDOABII585ltuFQ",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1706484048",
      "actor": {
        "login": "fanquake",
        "id": 863730,
        "node_id": "MDQ6VXNlcjg2MzczMA==",
        "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fanquake",
        "html_url": "https://github.com/fanquake",
        "followers_url": "https://api.github.com/users/fanquake/followers",
        "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
        "organizations_url": "https://api.github.com/users/fanquake/orgs",
        "repos_url": "https://api.github.com/users/fanquake/repos",
        "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fanquake/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2023-09-05T11:57:21Z",
      "updated_at": "2023-09-05T11:57:27Z",
      "author_association": "MEMBER",
      "body": "Sorry, can be ignored, that's actually a different test but still windows native CI continuing to fail randomly.",
      "user": {
        "login": "fanquake",
        "id": 863730,
        "node_id": "MDQ6VXNlcjg2MzczMA==",
        "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fanquake",
        "html_url": "https://github.com/fanquake",
        "followers_url": "https://api.github.com/users/fanquake/followers",
        "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
        "organizations_url": "https://api.github.com/users/fanquake/orgs",
        "repos_url": "https://api.github.com/users/fanquake/repos",
        "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fanquake/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/28221#issuecomment-1706484048",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221"
    },
    {
      "event": "commented",
      "id": 1706489431,
      "node_id": "IC_kwDOABII585ltvZX",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1706489431",
      "actor": {
        "login": "maflcko",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/maflcko",
        "html_url": "https://github.com/maflcko",
        "followers_url": "https://api.github.com/users/maflcko/followers",
        "following_url": "https://api.github.com/users/maflcko/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/maflcko/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/maflcko/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/maflcko/subscriptions",
        "organizations_url": "https://api.github.com/users/maflcko/orgs",
        "repos_url": "https://api.github.com/users/maflcko/repos",
        "events_url": "https://api.github.com/users/maflcko/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/maflcko/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2023-09-05T12:00:53Z",
      "updated_at": "2023-09-05T12:00:53Z",
      "author_association": "MEMBER",
      "body": "> Probably an upstream GCE outage?\r\n\r\nClosing for now. Let us know if this is still an issue.",
      "user": {
        "login": "maflcko",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/maflcko",
        "html_url": "https://github.com/maflcko",
        "followers_url": "https://api.github.com/users/maflcko/followers",
        "following_url": "https://api.github.com/users/maflcko/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/maflcko/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/maflcko/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/maflcko/subscriptions",
        "organizations_url": "https://api.github.com/users/maflcko/orgs",
        "repos_url": "https://api.github.com/users/maflcko/repos",
        "events_url": "https://api.github.com/users/maflcko/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/maflcko/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/28221#issuecomment-1706489431",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221"
    },
    {
      "event": "closed",
      "id": 10282778012,
      "node_id": "CE_lADOABII585th101zwAAAAJk5r2c",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/10282778012",
      "actor": {
        "login": "maflcko",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/maflcko",
        "html_url": "https://github.com/maflcko",
        "followers_url": "https://api.github.com/users/maflcko/followers",
        "following_url": "https://api.github.com/users/maflcko/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/maflcko/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/maflcko/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/maflcko/subscriptions",
        "organizations_url": "https://api.github.com/users/maflcko/orgs",
        "repos_url": "https://api.github.com/users/maflcko/repos",
        "events_url": "https://api.github.com/users/maflcko/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/maflcko/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2023-09-05T12:00:53Z"
    },
    {
      "event": "labeled",
      "id": 10282779562,
      "node_id": "LE_lADOABII585th101zwAAAAJk5sOq",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/10282779562",
      "actor": {
        "login": "maflcko",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/maflcko",
        "html_url": "https://github.com/maflcko",
        "followers_url": "https://api.github.com/users/maflcko/followers",
        "following_url": "https://api.github.com/users/maflcko/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/maflcko/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/maflcko/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/maflcko/subscriptions",
        "organizations_url": "https://api.github.com/users/maflcko/orgs",
        "repos_url": "https://api.github.com/users/maflcko/repos",
        "events_url": "https://api.github.com/users/maflcko/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/maflcko/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2023-09-05T12:01:01Z",
      "label": {
        "name": "Tests",
        "color": "d4c5f9"
      }
    },
    {
      "event": "commented",
      "id": 1858309126,
      "node_id": "IC_kwDOABII585uw4wG",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1858309126",
      "actor": {
        "login": "murchandamus",
        "id": 4060799,
        "node_id": "MDQ6VXNlcjQwNjA3OTk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/4060799?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/murchandamus",
        "html_url": "https://github.com/murchandamus",
        "followers_url": "https://api.github.com/users/murchandamus/followers",
        "following_url": "https://api.github.com/users/murchandamus/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/murchandamus/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/murchandamus/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/murchandamus/subscriptions",
        "organizations_url": "https://api.github.com/users/murchandamus/orgs",
        "repos_url": "https://api.github.com/users/murchandamus/repos",
        "events_url": "https://api.github.com/users/murchandamus/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/murchandamus/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2023-12-15T18:21:25Z",
      "updated_at": "2023-12-15T18:21:25Z",
      "author_association": "CONTRIBUTOR",
      "body": "The `wallet_transactiontime_rescan.py` test (and only it) frequently fails on my local machine due to timeout, whenever I run the tests with multiple threads, e.g.\r\n\r\n`test/functional/test_runner.py -j \"$(($(nproc)-1))\"`\r\n\r\nleads to \r\n\r\n```\r\n286/286 - wallet_transactiontime_rescan.py --legacy-wallet failed, Duration: 187 s\r\n\r\nstdout:\r\n2023-12-15T16:00:06.770000Z TestFramework (INFO): PRNG seed is: 5568797213703148493\r\n2023-12-15T16:00:06.771000Z TestFramework (INFO): Initializing test directory /tmp/test_runner_₿_🏃_20231215_105714/wallet_transactiontime_rescan_162\r\n2023-12-15T16:00:11.028000Z TestFramework (INFO): Prepare nodes and wallet\r\n2023-12-15T16:00:13.449000Z TestFramework (INFO): Start transactions\r\n2023-12-15T16:00:14.806000Z TestFramework (INFO): Send 10 btc to user\r\n2023-12-15T16:00:15.346000Z TestFramework (INFO): Send 5 btc to user\r\n2023-12-15T16:00:17.051000Z TestFramework (INFO): Send 1 btc to user\r\n2023-12-15T16:00:18.693000Z TestFramework (INFO): Check user's final balance and transaction count\r\n2023-12-15T16:00:18.695000Z TestFramework (INFO): Check transaction times\r\n2023-12-15T16:00:18.696000Z TestFramework (INFO): Restore user wallet on another node without rescan\r\n2023-12-15T16:00:19.221000Z TestFramework (INFO): Rescan last history part\r\n2023-12-15T16:00:19.232000Z TestFramework (INFO): Rescan all history\r\n2023-12-15T16:00:19.269000Z TestFramework (INFO): Check user's final balance and transaction count after restoration\r\n2023-12-15T16:00:19.271000Z TestFramework (INFO): Check transaction times after restoration\r\n2023-12-15T16:00:19.273000Z TestFramework (INFO): Test handling of invalid parameters for rescanblockchain\r\n2023-12-15T16:00:19.275000Z TestFramework (INFO): Test `rescanblockchain` fails when wallet is encrypted and locked\r\n2023-12-15T16:00:21.468000Z TestFramework (INFO): Test rescanning an encrypted wallet\r\n2023-12-15T16:02:13.710000Z TestFramework (ERROR): JSONRPC error\r\nTraceback (most recent call last):\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/authproxy.py\", line 151, in _get_response\r\n    http_response = self.__conn.getresponse()\r\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"/usr/lib/python3.11/http/client.py\", line 1378, in getresponse\r\n    response.begin()\r\n  File \"/usr/lib/python3.11/http/client.py\", line 318, in begin\r\n    version, status, reason = self._read_status()\r\n                              ^^^^^^^^^^^^^^^^^^^\r\n  File \"/usr/lib/python3.11/http/client.py\", line 279, in _read_status\r\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\r\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"/usr/lib/python3.11/socket.py\", line 706, in readinto\r\n    return self._sock.recv_into(b)\r\n           ^^^^^^^^^^^^^^^^^^^^^^^\r\nTimeoutError: timed out\r\n\r\nDuring handling of the above exception, another exception occurred:\r\n\r\nTraceback (most recent call last):\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/test_framework.py\", line 131, in main\r\n    self.run_test()\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/build/../test/functional/wallet_transactiontime_rescan.py\", line 221, in run_test\r\n    assert_equal(rescanning.result(), {\"start_height\": 0, \"stop_height\": 803})\r\n                 ^^^^^^^^^^^^^^^^^^^\r\n  File \"/usr/lib/python3.11/concurrent/futures/_base.py\", line 449, in result\r\n    return self.__get_result()\r\n           ^^^^^^^^^^^^^^^^^^^\r\n  File \"/usr/lib/python3.11/concurrent/futures/_base.py\", line 401, in __get_result\r\n    raise self._exception\r\n  File \"/usr/lib/python3.11/concurrent/futures/thread.py\", line 58, in run\r\n    result = self.fn(*self.args, **self.kwargs)\r\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/coverage.py\", line 50, in __call__\r\n    return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)\r\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/authproxy.py\", line 127, in __call__\r\n    response, status = self._request('POST', self.__url.path, postdata.encode('utf-8'))\r\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/authproxy.py\", line 106, in _request\r\n    return self._get_response()\r\n           ^^^^^^^^^^^^^^^^^^^^\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/authproxy.py\", line 153, in _get_response\r\n    raise JSONRPCException({\r\ntest_framework.authproxy.JSONRPCException: 'rescanblockchain' RPC took longer than 30.000000 seconds. Consider using larger timeout for calls that take longer to return. (-344)\r\n2023-12-15T16:02:13.765000Z TestFramework (INFO): Stopping nodes\r\n2023-12-15T16:02:13.765000Z TestFramework.node0 (ERROR): Unable to stop node.\r\nTraceback (most recent call last):\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/test_node.py\", line 369, in stop_node\r\n    self.stop(wait=wait)\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/coverage.py\", line 50, in __call__\r\n    return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)\r\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/authproxy.py\", line 127, in __call__\r\n    response, status = self._request('POST', self.__url.path, postdata.encode('utf-8'))\r\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/authproxy.py\", line 105, in _request\r\n    self.__conn.request(method, path, postdata, headers)\r\n  File \"/usr/lib/python3.11/http/client.py\", line 1286, in request\r\n    self._send_request(method, url, body, headers, encode_chunked)\r\n  File \"/usr/lib/python3.11/http/client.py\", line 1297, in _send_request\r\n    self.putrequest(method, url, **skips)\r\n  File \"/usr/lib/python3.11/http/client.py\", line 1123, in putrequest\r\n    raise CannotSendRequest(self.__state)\r\nhttp.client.CannotSendRequest: Request-sent\r\n2023-12-15T16:03:13.795000Z TestFramework.utils (ERROR): wait_until() failed. Predicate: ''''\r\n        wait_until_helper_internal(lambda: self.is_node_stopped(expected_ret_code=expected_ret_code, **kwargs), timeout=timeout, timeout_factor=self.timeout_factor)\r\n'''\r\n[node 2] Cleaning up leftover process\r\n[node 1] Cleaning up leftover process\r\n[node 0] Cleaning up leftover process\r\n\r\n\r\nstderr:\r\nTraceback (most recent call last):\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/build/../test/functional/wallet_transactiontime_rescan.py\", line 226, in <module>\r\n    TransactionTimeRescanTest().main()\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/test_framework.py\", line 154, in main\r\n    exit_code = self.shutdown()\r\n                ^^^^^^^^^^^^^^^\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/test_framework.py\", line 313, in shutdown\r\n    self.stop_nodes()\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/test_framework.py\", line 578, in stop_nodes\r\n    node.wait_until_stopped()\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/test_node.py\", line 417, in wait_until_stopped\r\n    wait_until_helper_internal(lambda: self.is_node_stopped(expected_ret_code=expected_ret_code, **kwargs), timeout=timeout, timeout_factor=self.timeout_factor)\r\n  File \"/home/murch/Workspace/2023-11-no-changeless-sffo/test/functional/test_framework/util.py\", line 276, in wait_until_helper_internal\r\n    raise AssertionError(\"Predicate {} not true after {} seconds\".format(predicate_source, timeout))\r\nAssertionError: Predicate ''''\r\n        wait_until_helper_internal(lambda: self.is_node_stopped(expected_ret_code=expected_ret_code, **kwargs), timeout=timeout, timeout_factor=self.timeout_factor)\r\n''' not true after 60 seconds\r\n```\r\n\r\nThe same error occurs on the latest `master` branch.\r\n\r\nOn either the test passes when I call it separately:\r\n\r\n```\r\n➜  build git:(master) ✗ ./test/functional/test_runner.py wallet_transactiontime_rescan\r\nTemporary test directory at /tmp/test_runner_₿_🏃_20231215_131217\r\nRunning Unit Tests for Test Framework Modules\r\n.....................\r\n----------------------------------------------------------------------\r\nRan 21 tests in 87.217s\r\n\r\nOK\r\nRemaining jobs: [wallet_transactiontime_rescan.py --descriptors, wallet_transactiontime_rescan.py --legacy-wallet]\r\n1/2 - wallet_transactiontime_rescan.py --descriptors passed, Duration: 20 s\r\nRemaining jobs: [wallet_transactiontime_rescan.py --legacy-wallet]\r\n2/2 - wallet_transactiontime_rescan.py --legacy-wallet passed, Duration: 51 s\r\n\r\nTEST                                             | STATUS    | DURATION\r\n\r\nwallet_transactiontime_rescan.py --descriptors   | ✓ Passed  | 20 s\r\nwallet_transactiontime_rescan.py --legacy-wallet | ✓ Passed  | 51 s\r\n\r\nALL                                              | ✓ Passed  | 71 s (accumulated) \r\nRuntime: 51 s\r\n```\r\n",
      "user": {
        "login": "murchandamus",
        "id": 4060799,
        "node_id": "MDQ6VXNlcjQwNjA3OTk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/4060799?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/murchandamus",
        "html_url": "https://github.com/murchandamus",
        "followers_url": "https://api.github.com/users/murchandamus/followers",
        "following_url": "https://api.github.com/users/murchandamus/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/murchandamus/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/murchandamus/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/murchandamus/subscriptions",
        "organizations_url": "https://api.github.com/users/murchandamus/orgs",
        "repos_url": "https://api.github.com/users/murchandamus/repos",
        "events_url": "https://api.github.com/users/murchandamus/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/murchandamus/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/28221#issuecomment-1858309126",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221"
    },
    {
      "event": "commented",
      "id": 1858812580,
      "node_id": "IC_kwDOABII585uyzqk",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1858812580",
      "actor": {
        "login": "maflcko",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/maflcko",
        "html_url": "https://github.com/maflcko",
        "followers_url": "https://api.github.com/users/maflcko/followers",
        "following_url": "https://api.github.com/users/maflcko/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/maflcko/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/maflcko/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/maflcko/subscriptions",
        "organizations_url": "https://api.github.com/users/maflcko/orgs",
        "repos_url": "https://api.github.com/users/maflcko/repos",
        "events_url": "https://api.github.com/users/maflcko/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/maflcko/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2023-12-16T12:57:40Z",
      "updated_at": "2023-12-16T12:57:40Z",
      "author_association": "MEMBER",
      "body": "Could bump the timeout inside this test to fix it?",
      "user": {
        "login": "maflcko",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/maflcko",
        "html_url": "https://github.com/maflcko",
        "followers_url": "https://api.github.com/users/maflcko/followers",
        "following_url": "https://api.github.com/users/maflcko/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/maflcko/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/maflcko/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/maflcko/subscriptions",
        "organizations_url": "https://api.github.com/users/maflcko/orgs",
        "repos_url": "https://api.github.com/users/maflcko/repos",
        "events_url": "https://api.github.com/users/maflcko/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/maflcko/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/28221#issuecomment-1858812580",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221"
    },
    {
      "event": "commented",
      "id": 1867929889,
      "node_id": "IC_kwDOABII585vVlkh",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1867929889",
      "actor": {
        "login": "murchandamus",
        "id": 4060799,
        "node_id": "MDQ6VXNlcjQwNjA3OTk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/4060799?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/murchandamus",
        "html_url": "https://github.com/murchandamus",
        "followers_url": "https://api.github.com/users/murchandamus/followers",
        "following_url": "https://api.github.com/users/murchandamus/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/murchandamus/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/murchandamus/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/murchandamus/subscriptions",
        "organizations_url": "https://api.github.com/users/murchandamus/orgs",
        "repos_url": "https://api.github.com/users/murchandamus/repos",
        "events_url": "https://api.github.com/users/murchandamus/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/murchandamus/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2023-12-22T17:39:09Z",
      "updated_at": "2023-12-22T17:39:09Z",
      "author_association": "CONTRIBUTOR",
      "body": "Well, if it’s just me that is affected that might be resolved by getting a faster computer.",
      "user": {
        "login": "murchandamus",
        "id": 4060799,
        "node_id": "MDQ6VXNlcjQwNjA3OTk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/4060799?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/murchandamus",
        "html_url": "https://github.com/murchandamus",
        "followers_url": "https://api.github.com/users/murchandamus/followers",
        "following_url": "https://api.github.com/users/murchandamus/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/murchandamus/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/murchandamus/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/murchandamus/subscriptions",
        "organizations_url": "https://api.github.com/users/murchandamus/orgs",
        "repos_url": "https://api.github.com/users/murchandamus/repos",
        "events_url": "https://api.github.com/users/murchandamus/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/murchandamus/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/28221#issuecomment-1867929889",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221"
    },
    {
      "event": "commented",
      "id": 1969064886,
      "node_id": "IC_kwDOABII5851XYu2",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1969064886",
      "actor": {
        "login": "m3dwards",
        "id": 1204616,
        "node_id": "MDQ6VXNlcjEyMDQ2MTY=",
        "avatar_url": "https://avatars.githubusercontent.com/u/1204616?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/m3dwards",
        "html_url": "https://github.com/m3dwards",
        "followers_url": "https://api.github.com/users/m3dwards/followers",
        "following_url": "https://api.github.com/users/m3dwards/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/m3dwards/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/m3dwards/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/m3dwards/subscriptions",
        "organizations_url": "https://api.github.com/users/m3dwards/orgs",
        "repos_url": "https://api.github.com/users/m3dwards/repos",
        "events_url": "https://api.github.com/users/m3dwards/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/m3dwards/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2024-02-28T14:09:47Z",
      "updated_at": "2024-02-28T14:09:47Z",
      "author_association": "CONTRIBUTOR",
      "body": "This test is also failing for me on Mac M1 also whenever I run with `--jobs`  greater than 1.\r\n\r\nWill look into it.",
      "user": {
        "login": "m3dwards",
        "id": 1204616,
        "node_id": "MDQ6VXNlcjEyMDQ2MTY=",
        "avatar_url": "https://avatars.githubusercontent.com/u/1204616?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/m3dwards",
        "html_url": "https://github.com/m3dwards",
        "followers_url": "https://api.github.com/users/m3dwards/followers",
        "following_url": "https://api.github.com/users/m3dwards/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/m3dwards/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/m3dwards/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/m3dwards/subscriptions",
        "organizations_url": "https://api.github.com/users/m3dwards/orgs",
        "repos_url": "https://api.github.com/users/m3dwards/repos",
        "events_url": "https://api.github.com/users/m3dwards/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/m3dwards/received_events",
        "type": "User",
        "site_admin": false,
        "patch_url": null
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/28221#issuecomment-1969064886",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/28221"
    },
    {
      "event": "locked",
      "id": 16492008784,
      "node_id": "LOE_lADOABII585th101zwAAAAPXABVQ",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/16492008784",
      "actor": {
        "login": "bitcoin",
        "id": 528860,
        "node_id": "MDEyOk9yZ2FuaXphdGlvbjUyODg2MA==",
        "avatar_url": "https://avatars.githubusercontent.com/u/528860?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/bitcoin",
        "html_url": "https://github.com/bitcoin",
        "followers_url": "https://api.github.com/users/bitcoin/followers",
        "following_url": "https://api.github.com/users/bitcoin/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/bitcoin/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/bitcoin/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/bitcoin/subscriptions",
        "organizations_url": "https://api.github.com/users/bitcoin/orgs",
        "repos_url": "https://api.github.com/users/bitcoin/repos",
        "events_url": "https://api.github.com/users/bitcoin/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/bitcoin/received_events",
        "type": "Organization",
        "site_admin": false,
        "patch_url": null
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2025-02-27T07:34:53Z"
    }
  ]
}